TLM/TLM/State/Options.cs

```diff
@@ -387,6 +387,24 @@ public class Options : MonoBehaviour {
 			enableLaneConnectorToggle = featureGroup.AddCheckbox(Translation.GetString("Lane_connector"), laneConnectorEnabled, onLaneConnectorEnabledChanged) as UICheckBox;
 
 			Indent(turnOnRedEnabledToggle);
+
+                        // KEYBOARD
+                        ++tabIndex;
+
+                        AddOptionTab(tabStrip, Translation.GetString("Keyboard"));
```

Comment:
Maybe I'm missing something, but the "Keyboard" translation was not added to the list. Also, same suggestion here, rename to "Keybinds."

Reply:
Done